CONTRIBUTING.md

```diff
@@ -45,6 +45,41 @@ Before you can start contributing to the CyberArk Secrets Provider for Kubernete
     
     b. For detailed setup instructions for OSS, see [CyberArk Secrets Provider for Kubernetes for OSS](https://docs.conjur.org/Latest/en/Content/Integrations/Kubernetes_deployApplicationsConjur-k8s-Secrets.htm).
     
+### Deploy a Local Dev Environment
+
+You can now deploy a local development environment for Kubernetes using [Docker Desktop](https://www.docker.com/products/docker-desktop). Docker Desktop provides a convenient way to deploy and develop from your machine against a cluster deployed locally. 
+To enable this, perform the following:
+
+1. Navigate to `bootstrap.env` and uncomment the `Local DEV Env` section, ensuring that `DEV=true`
+
+1. Run `./bin/start --dev --gke`, appending `--oss` or `--dap` according to the environment that needs to be deployed 
+
+1. If a cluster is already locally deployed, run `./bin/start --dev --reload`. 
+The reload script will build your local changes and redeploy them to the local Secrets Provider K8s cluster.
+
+Currently the development environment only supports deployment to Kubernetes and *not* Openshift.
+
+#### Helpful hints
+                  
+This Secrets Provider for K8s uses the [Kubernetes Conjur deploy](https://github.com/cyberark/kubernetes-conjur-deploy/blob/master/CONTRIBUTING.md) repository to deploy Conjur/DAP on Kubernetes. 
+By default, 2.0 Gib of memory is allocated to Docker on your computer. 
+
+To successfully deploy a DAP cluster (Master + Followers + Standbys), you will need to increase the memory limit to 6 Gib. To do so, perform the following:
+
+1. Navigate to Docker preferences
+
+1. Click on "Advanced" and slide the "Memory" bar to 6
+
+Before deploying locally using Docker Desktop, ensure you are in the proper docker-desktop context. Otherwise, the deployment will not run successfully.
+
+To switch to a local context: `kubectl config use-context docker-desktop`
+
+Note that if you decide to deploy against a remote K8s cluster, you will need to update your context to point to our GKE cluster.
```

Comment:
this should not be a helpful hint but a step as it's required. I also suggest adding that before switching the context they should run `kubectl config current-context` to show which context they are currently on so they can switch back to it easily

Reply:
also - this didn't work for me without enabling k8s

Reply:
this can be useful: `current_context=$(kubectl config current-context)`